daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -18,6 +18,25 @@
 FAIL=0
 FAILURES=""
 
+function wait_for_connectivity {
```

Comment:
Do we need to do this for Centos 6 translate as well (or is that already fixed)?

Reply:
I haven't seen any network errors in the translation script, so I'm fine waiting for the root-cause fix that Liam is working on. But if we see errors, we can rethink that.